<launch>

    <rosparam command="load" file="$(find ttc_object_avoidance)/params/tracker.yaml"/>
    <rosparam command="load" file="$(find ttc_object_avoidance)/params/airsim_recording.yaml"/>
    <node name="feature_tracker" pkg="ttc_object_avoidance" type="feature_tracker" output="screen"/>

    <node name="displacement_tracker" pkg="ttc_object_avoidance" type="displacement_tracker_node.py" output="screen"/>

    <node name="airsim_recording_reader" pkg="ttc_object_avoidance" type="airsim_recording_reader_node.py" output="screen"/>

    <!-- <node name="area_tracker" pkg="ttc_object_avoidance" type="area_tracker_node.py" output="screen"/> -->

</launch>